Use sRGB profile from ArgyllCMS

davidmz opened the pull request:
Added test to check thumbnail color when original image is not in sRGB. Closes T85.

clbn commented on test/unit/attachment.js:
```diff
@@ -249,5 +259,39 @@ describe('Attachment', function() {
+          h: 350,
+          url: config.attachments.url + config.attachments.imageSizes.t2.path + newAttachment.id + '.' + newAttachment.fileExtension
+        }
+      })
```
I'd prefer to remove lines 269-286, for the test clarity.

davidmz committed:
Use sRGB profile from ArgyllCMS

Added test to check thumbnail color when original image is not in sRGB. Closes T85.
